assets/css/modal_prompt.css

```diff
@@ -0,0 +1,16 @@
+.modal_prompt--container {
+  background-color: #d8d8d8;
+  border-radius: 4px;
+  box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.2);
```

Comment:
Not sure what the general principle is, but should this and `border-radius` be in `rem` or `em` instead of `px`?

Reply:
I think I got these from the Abstract mock-ups "inspect" section. But, yeah, since we try to size things in `rem` generally, I'll change it to that.